examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -113,24 +104,31 @@ echo "submitting script to do compilation"
 sed -i s/\<NAME\>/standalone/g compile.daint.slurm
 sed -i s/\<NTASKS\>/$ranks/g compile.daint.slurm
 sed -i s/\<NTASKSPERNODE\>/1/g compile.daint.slurm
-sed -i s/\<CPUSPERTASK\>/$nthreads/g compile.daint.slurm
+sed -i s/\<CPUSPERTASK\>/$NTHREADS/g compile.daint.slurm
```

Comment:
I don't mind the change, but what is the motivation for the change? nthreads could be set the buildenv if we ever use that here (and we do use it to get the slurm script templates)

Reply:
Yes. The integration with `jenkins.sh` and buildenv was left for a future PR. I just moved this to the configuration section to make it more visible.

Reply:
I more mean the renaming of lower case nthreads to allcaps NTHREADS? I saw the jenkins integration is for  future PR, but why use a different name? not a huge thing, just curious

Reply:
oh, 'to be more visible', you already said, got it :)